python/clx/tests/test_phish_detector.py

```diff
@@ -31,26 +33,42 @@ def test_train_model():
         fake = Faker()
         email_col = [fake.text() for _ in range(200)]
         label_col = [random.randint(0, 1) for _ in range(200)]
-        emails_pd = pd.DataFrame(list(zip(email_col, label_col)), columns=["email", "label"])
+        emails_pd = pd.DataFrame(
```

Comment:
Do we need to create this intermediary Pandas DF? Can't we just construct the cuDF directly from the zipped list?

Reply:
yes we can do that. thx for the catch.